Compentio.SourceMapper.Generator/Processors/InterfaceProcessorStrategy.cs

```diff
@@ -65,7 +65,7 @@ private string GenerateInterfaceMethods(IMapperMetadata sourceMetadata)
 
         private string GenerateInterfaceMethod(IMethodMetadata methodMetadata)
         {
-            return $"{AttributesMatchers.GetInverseMethodFullName(methodMetadata)};";
+            return $"{GetInverseMethodFullName(methodMetadata)};";
```

Comment:
It seems to be like methodMetedata property

Reply:
Updated.